TypeScript.YAML-tmLanguage

```diff
@@ -339,7 +339,7 @@ repository:
     name: meta.type.function.return.ts
     begin: '=>'
     beginCaptures:
-      '0': { name: keyword.others.ts }
+      '0': { name: keyword.operator.ts }
```

Comment:
There was no field as keyword.operand. And then he said -- keep them as --. 
Considering the definitions of operator and operand I felt he probably meant operator, which they were earlier.